src/Domain.LinnApps/Pcas/PcasStorageType.cs

```diff
@@ -0,0 +1,53 @@
+﻿namespace Linn.Stores2.Domain.LinnApps.Pcas
+{
+    using Linn.Stores2.Domain.LinnApps.Stock;
+
+    public class PcasStorageType
+    {
+        public PcasStorageType(
+            string boardCode,
+            string storageTypeCode,
```

Comment:
refactor this to pass in a PcasBoard and a StorageType as oppsoed to just to strings, otherwise EF Core will try to INSERT new boards and storage types instead of correctly linking to existing ones